### ISSUE TITLE
Add prominence to Comment

### ISSUE THREAD

**Issue by garethrees** [mysociety/alaveteli#5423]
Currently comments use a Boolean `visible` flag.

Allowing `visible`, `requester_only` or `hidden` would make annotations more applicable for many use cases.

For example, Pro users could use `requester_only` comments to add personal notes (https://github.com/mysociety/alaveteli-professional/issues/301), or admins could leave `hidden` comments on requests for other admins (https://github.com/mysociety/alaveteli/issues/5422).

We'd have to consider who gets notified depending on visibility.

**Comment by RichardTaylor** [comment]
We have a policy of running WhatDoTheyKnow.com in as transparent manner as we can. 

If we hide an annotation there is no record on the request thread itsself that we have done so.

On occasion it might be better to have a public prominence reason and show that an annotation has been removed from public view. 

Often annotations we are removing are spam/junk so removing them and stopping them cluttering up the request page is often appropriate. 

I note the public event log history for a request thread includes annotation creation / hiding so there is some public transparency there for those who dig for it. 

We at WhatDoTheyKnow have a policy of trying not to spend too much admin time on annotations as they are not the core of the service - we want to focus our efforts on requests/responses; following from that I think it's right that we generally make very clear when we remove messages from correspondence threads, but it's harder to see that we've removed an annotation/comment. 

Overall - I think what we are doing is fine, but there are some potential benefits in terms of being transparent about takedowns which could arise from treating annotations as we do messages on threads when it comes to prominence / prominence reasons.
